style.css

```diff
@@ -10,12 +10,43 @@ body {
   overflow-x: hidden;
 }
 
+.nav {
+  z-index: 10;
+}
+
+.hamburger.hide {
+  display: none;
+}
+
+.menu_mobile.active {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  gap: 20px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  padding: 100px;
+  backdrop-filter: blur(8px);
+}
```

Comment:
- [ ] Kindly make sure when the mobile menu is opened it fits the whole height visible screen and make sure to give the container of the menu the background image in order to match the provided Figma design as much as possible.

- [ ] KIndly make sure the navigation links have the same typography( `font, colors, font-size, font-weight, etc...`) as the provided design. Please make sure each nav link has an underline style.


   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/232700616-911e0fb1-f5ee-47b3-a4e0-b07e526bf40f.png)

   - ### FIGMA DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/232700743-311ef9fa-4cc2-4ef5-892e-5366a2a7e6c4.png)